src/i18n/locales/en.ts

```diff
@@ -722,6 +722,8 @@ export const en = {
     clear_layout: 'Clear layout',
     add_another_multiview: 'Add another multiview',
     could_not_delete_layout: 'Could not delete layout',
+    could_not_delete_layout_in_use:
+      'Could not delete layout that is saved in production',
```

Comment:
Not sure about whether 'Could' or 'Can' is the correct word... Possibly: 'The layout is being used and can not be deleted' (similar to the Swedish text)